Python/cp/Number_of_K-Length_Sublists_with_Average_at_Least _Target.py

```diff
@@ -28,11 +28,14 @@
 if (p//k)>=target: #verifying the taken avg sum greater than target or not
```

Comment:
You have been doing great! However, a small correction can be made. Always make sure to add comments above the code segment and never alongside. Can you please correct it and request it again with all the modifications. That would be appreciated :)

Reply:
@bhav09 sir changes are done